indra/assemblers/pybel_assembler.py

```diff
@@ -0,0 +1,155 @@
+                # Define the enzyme node
+                (enz_node, enz_attr) = _get_agent_node(stmt.enz)
+                (sub_node, sub_attr) = _get_agent_node(stmt.sub)
+                sub_attr[pc.VARIANTS] = [
+                                    {pc.KIND: pc.PMOD,
+                                     pc.IDENTIFIER: {
+                                       pc.NAMESPACE: pc.BEL_DEFAULT_NAMESPACE,
+                                       pc.NAME: 'Ph'},
+                                     PmodParser.CODE: stmt.residue,
+                                     PmodParser.POSITION: stmt.position }]
+                self.model.add_node(enz_node, enz_attr)
+                self.model.add_node(sub_node, sub_attr)
+                # Create an edge for each evidence object
+                edge = {pc.SUBJECT: {
+                        pc.EFFECT: {pc.NAME: 'kin',
+                                    pc.NAMESPACE: pc.BEL_DEFAULT_NAMESPACE},
+                        pc.MODIFIER: pc.ACTIVITY},
+                        pc.RELATION: pc.INCREASES,
+                        pc.OBJECT: {}}
+                        #'object': {
+                        #    'effect': {'name': 'kin', 'namespace': 'bel'},
+                        #    'modifier': 'Activity'}}
+                # If there's no evidence for this statement, add node without
+                # any evidence info
+                import ipdb; ipdb.set_trace()
+                if not stmt.evidence:
+                    edge[pc.ANNOTATIONS] = {}
+                    edge[pc.CITATION] = {}
+                    edge[pc.EVIDENCE] = ''
+                    self.model.add_edge(enz_node, sub_node, attr_dict=edge)
+                # Otherwise, add an edge for each piece of evidence.
+                else:
+                    for ev in stmt.evidence:
+                        edge[pc.ANNOTATIONS] = {}
+                        # FIXME Retrieve citation information from pubmed_client
+                        edge[pc.CITATION] = {'authors': '', 'comments': '',
+                                             'date': '', 'name': '',
+                                             'reference': ev.pmid,
+                                             'type': 'PubMed'},
+                        edge[pc.EVIDENCE] = ev.text
+                        self.model.add_edge(enz_node, sub_node, attr_dict=edge)
+        return self.model
+
+def _get_agent_node(agent):
+    if agent is None:
+        return (None, None)
+    # FIXME: For now, deal only with agents having HGNC grounding
+    # FIXME: For now, don't look at bound/mod/mut/loc/activity conditions
+    hgnc_id = agent.db_refs.get('HGNC')
+    if hgnc_id is None:
+        return (None, None)
+    hgnc_name = hgnc_client.get_hgnc_name(hgnc_id)
+    node = (pc.PROTEIN, 'HGNC', hgnc_name)
+    node_attr = {pc.FUNCTION: pc.PROTEIN, pc.NAMESPACE: 'HGNC',
+                 pc.NAME: hgnc_name}
+    return (node, node_attr)
+
+def _get_citation(evidence):
+    pass
+
+"""
```

Comment:
Great start! I can either provide you examples on what to put where, or you could dive into reading the [unit tests](https://github.com/pybel/pybel/blob/develop/tests/test_parse_bel.py) in PyBEL. They're all structured the same so it would be easy to look up what tuple/data dictionary go with which BEL nodes.